mhs-reference-implementation/mhs/SDSLookup/tests/test_sdsclient.py

```diff
@@ -0,0 +1,68 @@
+from unittest import TestCase
+import mhs.SDSLookup.sds as sds
+import mhs.SDSLookup.tests.ldap_mocks as mocks
+from utilities.test_utilities import async_test
+
+NHS_SERVICES_BASE = "ou=services, o=nhs"
+
+MHS_OBJECT_CLASS = "nhsMhs"
+
+PARTY_KEY = "A91461-9199084"
+SERVICE_ID = "urn:nhs:names:services:psis:MCCI_IN010000UK13"
+ODS_CODE = "A91461"
+
+
+class TestSDSClient(TestCase):
+
+    @async_test
+    async def test_mhs_details_lookup(self):
+        server, connection = mocks.fake_ldap_server()
```

Comment:
You do this setup step several times. Might be worth extracting it to a method.

Reply:
I'd actually already built a method that does all this in the ldap mocks but forgot to use it 🤦‍♂